client/src/components/Tickets.vue

```diff
@@ -5,7 +5,8 @@
       <ul>
         <li v-for="ticket in tickets">
           <div class="ticket" @click="voteOn(ticket)">
-            <h4 :class="{ voting : ticket.votingOn }">{{ ticket.name }} <span v-if="ticket.score">{{ ticket.score }}</span></h4>
+            <h4 :class="{ voting: ticket.votingOn }">{{ ticket.name }} <span v-if="ticket.score">{{ ticket.score
```

Comment:
there's an issue displaying more than one letter in the score of the ticket:
![image](https://github.com/LukeGarrigan/planfree.dev/assets/12545967/4a5de81f-3da1-45df-ae71-5752a1e6ab1a)